app/scripts/controllers/eventDetails.js

```diff
@@ -655,11 +665,27 @@ angular
         return currentMinistry ? currentMinistry.activities : [];
       };
 
+      $scope.getEventTypes = () => {
+        const currentMinistry =
+          $scope.ministries &&
+          $scope.ministries.find((m) => m.id === $scope.conference.ministry);
+        const currentPurpose =
+          $scope.ministryPurposes &&
+          $scope.ministryPurposes.find((m) => m.id === $scope.conference.type);
+        return currentMinistry.eventTypes &&
+          currentPurpose &&
+          ((currentPurpose.name && currentPurpose.name.includes('Mission')) ||
+            currentPurpose.name.includes('Conference'))
+          ? currentMinistry.eventTypes
+          : [];
```

Comment:
I'm unsure if this is correct. Currently the event type field is only showing when a `Ministry Hosting Event` selected which has `eventTypes` and a `Ministry Purpose` was selected

Reply:
We had a convo about this in the comments of the Jira ticket so now it only shows when certain Ministry Purposes are also selected.